packages/usa-in-page-navigation/src/test/in-page-nav-custom-content.spec.js

```diff
@@ -0,0 +1,67 @@
+const assert = require("assert");
+const fs = require("fs");
+const path = require("path");
+const sinon = require("sinon");
+const behavior = require("../index");
+
+const TEMPLATE = fs.readFileSync(
+  path.join(__dirname, "/in-page-nav-custom-content.template.html")
+);
+const PRIMARY_CONTENT_SELECTOR =
+  ".usa-in-page-nav-container .usa-in-page-nav .usa-in-page-nav__list";
+
+const tests = [
+  { name: "document.body", selector: () => document.body },
+  {
+    name: "in page nav",
+    selector: () => document.querySelector(".usa-in-page-nav"),
+  },
+];
+
+tests.forEach(({ name, selector: containerSelector }) => {
+  describe(`in-page navigation initialized at ${name}`, () => {
+    const { body } = document;
+
+    let theList;
+    let listLinks;
+
+    before(() => {
+      const observe = sinon.spy();
+      const mockIntersectionObserver = sinon.stub().returns({ observe });
+      window.IntersectionObserver = mockIntersectionObserver;
+    });
+
+    beforeEach(() => {
+      body.innerHTML = TEMPLATE;
+
+      behavior.on(containerSelector());
+
+      theList = document.querySelector(PRIMARY_CONTENT_SELECTOR);
+      listLinks = Array.from(theList.getElementsByTagName("a"));
+    });
+
+    afterEach(() => {
+      behavior.off(containerSelector(body));
+      body.innerHTML = "";
+      window.location.hash = "";
+    });
+
+    it("creates only one link item in the nav list", () => {
+      assert.equal(listLinks.length === 1, true);
+    });
+
+    it("creates a link in the nav list for the header that is inside the custom content region", () => {
+      const customRegionLink = listLinks.filter((link) =>
+        link.href.includes("#header-in-content-region")
+      );
+      assert.equal(customRegionLink.length === 1, true);
```

Comment:
Instead of updating the test highlighted here, I updated the first unit test that checked for exactly 1 link in the link list in [this commit](https://github.com/uswds/uswds/pull/5387/commits/394a4638a6b054211002f90a3c87c3a809d29c67#diff-480971ade490350a6945fd11e94390f4a9a9112085a411450aa831fea612c1c4L49-R55). I might have misunderstood the request, so let me know if wanted me to change this one instead.